cloudformation.yaml

```diff
@@ -225,15 +259,32 @@ Resources:
                  "Next": "MigrateShards",
                  "Retry": [ {
                    "ErrorEquals": [ "States.ALL" ],
-                   "IntervalSeconds": 10,
-                   "MaxAttempts": 12
+                   "IntervalSeconds": 30,
+                   "MaxAttempts": 20,
+                   "BackoffRate": 1.0
```

Comment:
The default is to increase the interval [each time](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-error-handling.html):

> The multiplier by which the retry interval increases during each attempt (2.0 by default).

But I don't really want to back off in this case.